bcgov/wps
# Nats openshift yaml and deployment script

To test:
0. Install `nats-cli`:
```
brew tap nats-io/nats-tools
brew install nats-io/nats-tools/nats
```
1. Run  `oc -n e1e498-dev port-forward nats-0 4222:4222` to message yourself
2. In another cli tab run `nats sub cli.demo`
3. In another cli table run `nats pub cli.demo "message {{.Count}} @ {{.TimeStamp}}" --count=10`
4. Watch the tab that subscribes to see all the wonderful messages you've sent to the subscription `cli.demo`

# Test Links:
[Percentile Calculator](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2215.apps.silver.devops.gov.bc.ca/fwi-calculator)
